src/main/daml/Daml/Finance/Interface/Account/Account.daml

```diff
@@ -98,7 +96,7 @@ interface Account where
     with
       quantity : Quantity InstrumentKey Decimal
         -- ^ The target `Instrument` and corresponding amount.
-    controller (view this).custodian, (view this).owner
+    controller (view this).custodian, (view this).controllers.approvers
```

Comment:
One thought / suggestions:

We could rename `approvers` to `creditControllers`, and `instructors` to `debitControllers`. I think those are only used in this context here, and it would make it much more explicit what they are actually used for.

Wdyt?

Reply:
I like that

Reply:
We would probably need to update quickstarter + docs with the renaming. I still have to figure out if we anyway need to update the quickstarter with this PR, so maybe the renaming comes at "zero cost"

Reply:
actually it is not `creditControllers` and `debitControllers` but rather parties that is controlling incoming and outgoing transfers. the controllers of 

1. `Credit` is `custodian` and `controllers.approvers`  
2. `Debit` is `custodian` and `controllers.instructors`

So, both choices has the `custodian` as controller independent of the `controllers`. That is mainly to facilitate for the `custodian` to preserve quantities (e.g., in case the instructors and approvers is owner-only, we don't want them to simply debit or credit out of context).

Reply:
I see - and we can't just say the `custodian` should be part of the `controllers.*` sets because we want to support an owner-transferable holding, while the account credits and debits are still authorized by the custodian (and an owner-controlled credit/debit doesn't make much sense).

So maybe it's `incomingControllers` and `outgoingControllers`? I just find the `instructors` notion confusing (does it even relate to the instructors for settlement?), and to some extend the `approvers` also don't quite describe what they're used for.

Reply:
btw, happy for such renaming to go into a separate PR (if we can agree on something sensible), don't need to resolve it here.

Reply:
yep, like that. will change accordingly

Reply:
> btw, happy for such renaming to go into a separate PR (if we can agree on something sensible), don't need to resolve it here.

had anyway to revert the previous renaming.